products/sentinel_one.py

```diff
@@ -176,7 +188,7 @@ def _get_default_body(self) -> dict:
         """
         Get the default request body for a SentinelOne API query.
         """
-        return {"siteIds": [self._site_id]} if self._site_id else {"accountIds": [self._account_id]}
+        return {"siteIds": self._site_ids} if self._site_ids else {"accountIds": [self._account_ids]}
```

Comment:
```suggestion
        return {"siteIds": self._site_ids} if self._site_ids else {"accountIds": self._account_ids}
```
remove brackets to prevent type mismatch when performing DV query in `_process_queries` function